src/utils/cliHelper.js

```diff
@@ -25,6 +26,14 @@ class CLIHelper {
     ;[this.config.output, this.config.source]
       .filter(dir => !dirExist(dir))
       .forEach(dir => errors.push(`${dir} folder does not exist`))
+    ;[
+      this.config.ignore,
+      this.config.ignoreDestructive,
+      this.config.include,
+      this.config.includeDestructive,
+    ]
+      .filter(file => file && !fileExist(file))
+      .forEach(file => errors.push(`${file} file does not exist`))
```

Comment:
**praises:** nice touch here, it was not enforced before, so there was no feedback for the end user
**suggestion(if-minor):** add unit test for those

Reply:
👍